summarization/bad_request_error.py

```diff
@@ -0,0 +1,10 @@
+'''Module to define BadRquestError'''
+
+
+class BadRequestError(Exception):
+    '''Exception raised when the API call was not successfully'''
```

Comment:
Did you mean "completed successfully" here ?

Reply:
Ah yes thanks for catching this!

Reply:
Updated accordingly!